src/modules/News/controller.ts

```diff
@@ -78,4 +78,32 @@ const getTopHeadlines = async (req: Request, res: Response) => {
   }
 };
 
-export default { getLatestNews, getTopHeadlines, getPersonalizedNews };
+const fetchSingleArticle = async (req: Request, res: Response) => {
```

Comment:
I would change this name to getSingleArticle to keep the pattern to other controllers.

Reply:
using fetch top make a disctintion between getting from our own api and fetching from news api